docs/getting-started/quick-start.md

```diff
@@ -4,12 +4,100 @@ sidebar_position: 2
 
 # Quick Start
 
-In this section we are providing a sample application that one can deploy and just use to play around.
+In this quick start, you will see a sample application that you can deploy and
+use to play around.  As part of this exercise, a backing service and an
+application is required.  The backing service is a PostgreSQL database and
+application is a [Spring Boot REST API server][petclinic].
 
-## K8S
+## Database Backend
 
-_Here we provided a small sample on K8s_
+You can install [Crunchy PostgreSQL from OperatorHub.io][crunchy].  After the
+installation, to create a PostgreSQL cluster, run this command:
 
-## OpenShift
+```bash
+kubectl apply -f https://gist.github.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/04eb5fe3d7f393af5a6760b03d9a1a3f5c725077/pgcluster.yaml
+```
 
-_Here we provided a small sample on OpenShift_
\ No newline at end of file
+Ensure all the pods in `petclinic-demo` is running (it will take few minutes):
+
+```bash
+kubectl get pod -n petclinic-demo
+```
+
+You should see output something like this:
+
+```
+NAME                                          READY   STATUS    RESTARTS   AGE
+backrest-backup-hippo-9gtqf                   1/1     Running   0          13s
+hippo-597dd64d66-4ztww                        1/1     Running   0          3m33s
+hippo-backrest-shared-repo-66ddc6cf77-sjgqp   1/1     Running   0          4m27s
+```
+
+Now you can initialize the database with this command:
```

Comment:
similarly, here, we don't know what are the "special" things you have in this script.
I think we should detail what the script is doing.